erpnext/hr/doctype/expense_claim/expense_claim.py

```diff
@@ -36,6 +36,10 @@ def validate(self):
 		if self.task and not self.project:
 			self.project = frappe.db.get_value("Task", self.task, "project")
 
+	def on_update(self):
+		if self.exp_approver and self.approval_status == "Draft":
+			self.notify_expense_claim_approver()
```

Comment:
So, this email will go on every save of the Expense Claim, which is not so good. Add a button instead to send the email.